### PR TITLE
test: added integration test boilerplate for get_refresh_token_scope

### LINKED ISSUE #69: Integraion test for get_scopes_from_refresh_token token.
An integration test for `get_scopes_from_refresh_token` with the actual live server is needed. 

**DoD**
- fetch refresh token from the live server using **Cloud KMS.**
- Use the token to extract its scope
- assert if the scopes are the same as expected scopes
-  In the CI pipeline we will be using our `test_project` 

**Challenges**
-  Requires a credit card to create a GCP account, hence to implement this you must have a credit card account.
- Enabling Cloud KMS const as low as **3.00 USD**

### PR DESCRIPTION
**Pull request by sudiptob2**
Relate to #69 

Progress

- [x] added boilerplate for testing `get_scopes_from_refresh_token`

Todo

- [ ] implement a way to get credentials inside tests
- [ ] update `test/refresh_scopes` file

### CODE REVIEW

File: src/gcp_scanner/test_unit.py [hunk 1]
```diff
@@ -208,6 +208,30 @@ def test_get_scope_from_rt_no_scope(self, mocked_post):
     )
 
 
+class TestScopesIntegration(unittest.TestCase):
+  """Integration test against live test-porject."""
```
Comment (mshudrak) [inline src/gcp_scanner/test_unit.py:212]: porject/project

File: src/gcp_scanner/test_unit.py [hunk 2]
```diff
@@ -208,6 +208,30 @@ def test_get_scope_from_rt_no_scope(self, mocked_post):
...
+  def setUp(self):
+    # TODO: get_creds_from_metadata or some other method should
```
Comment (mshudrak) [inline src/gcp_scanner/test_unit.py:214]: May be open an issue describing what need to be done and reference it here?

Comment (mshudrak) [reply src/gcp_scanner/test_unit.py:214]: This way we can have more context.

**Review by mshudrak — CHANGES_REQUESTED** [review]
Small comments. Overall looks good.

File: src/gcp_scanner/test_unit.py (hunk 2 above)
Comment (sudiptob2) [reply src/gcp_scanner/test_unit.py:214]: I have added this issue #69 for this, should we create another one ?

Comment (mshudrak) [reply src/gcp_scanner/test_unit.py:214]: Ah, I see my bad. Could you please reference the issue in the code? e.g. i#69

**Review by mshudrak — CHANGES_REQUESTED** [review]
Thanks for the quick fix. Right now the test is failing. Let's have the test but let's avoid failing for now.

File: src/gcp_scanner/test_unit.py (hunk 2 above)
Comment (sudiptob2) [reply src/gcp_scanner/test_unit.py:214]: sure

**Review by mshudrak — APPROVED** [review]

**Comment by mshudrak** [comment]
Thanks, merging.

**Pull request merged**